tests/routers/test_score_set.py

```diff
@@ -98,6 +98,7 @@ def test_create_minimal_score_set(client, setup_router_db):
     expected_response = update_expected_response_for_created_resources(
         deepcopy(TEST_MINIMAL_SEQ_SCORESET_RESPONSE), experiment, response_data
     )
+    expected_response["experiment"].update({"numScoreSets": 1})
```

Comment:
Would it make more sense to add a property to the expected response constant in `tests.helpers.constants` with this property so that we don't have to update the expected response each time?

Reply:
I considered this one before. It's a dynamic value, especially when testing the permission, whether private or public, for contributors and superseding cases. That's why I finally don't put it in `tests.helpers.constants`. Even though we put it in `tests.helpers.constants`, most of the time we still need to update it manually. I modify this part to set it to 0 in `tests.helpers.constants`, and manually update it on the rest tests. I don't have any other better solution for this part.